Dockerfile

```diff
@@ -1,34 +1,41 @@
-FROM ubuntu:20.04 as base
+## Stage 1: Build indy-sdk and postgres plugin
 
-ENV DEBIAN_FRONTEND noninteractive
+FROM ubuntu:22.04 as base
```

Comment:
I'm in favour of using the newer ubuntu version. However, due to indy-sdk we need to rely on the downgraded libssl 1.1 packages.  Do you guys think this could represent an issue?

Reply:
I think it is fine for now. We should probably update this Dockerfile to use Askar instead of indy-sdk by default